docs/index.rst

```diff
@@ -2,7 +2,7 @@
 Welcome to ReFrame
 ==================
 
-ReFrame is a new framework for writing regression tests for HPC systems.
+ReFrame is a framework for writing tests for HPC systems that can check for functionality and performance regressions.
```

Comment:
```suggestion
ReFrame is a framework for writing tests targeting HPC systems that can check for functionality and performance regressions.
```

Reply:
I've changed the whole sentence to "ReFrame is a high-level framework for writing regression tests for HPC systems."